pkg/queue/event/sender.go

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2022-2023 Exactpro (Exactpro Systems Limited)
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package event
+
+import p_buff "th2-grpc/th2_grpc_common"
+
+type BatchSender interface {
```

Comment:
May BatchSender and BatchSubscriber should be private interfaces

Reply:
We don't even need them. I will remove those interfaces for messages and events